landoapi/models/treestatus.py

```diff
@@ -53,21 +52,23 @@ def is_open(self) -> bool:
         the appropriate Phabricator group review for approval (`release-managers`)
         and the hg hook will enforce `a=<reviewer>` is present in the commit message.
         """
-        return self != TreeStatus.CLOSED
+        return self in {TreeStatus.OPEN, TreeStatus.APPROVAL_REQUIRED}
 
 
-DEFAULT_TREE = {
-    "category": TreeCategory.OTHER,
-    "reason": "New tree",
-    "status": TreeStatus.CLOSED,
-    "tags": [],
-    "log_id": None,
-}
+def get_default_tree():
+    return {
+        "category": TreeCategory.OTHER,
+        "reason": "New tree",
+        "status": TreeStatus.CLOSED,
+        "tags": [],
+        "log_id": None,
+    }
 
 
-def load_last_state(last_state_str: str) -> dict:
+def load_last_state(last_state_orig: dict) -> dict:
     """Ensure that structure of last_state is backwards compatible."""
```

Comment:
nit: could add more context here as to what it should be backwards compatible with.